sdk/storage/azure-storage-files-shares/CHANGELOG.md

```diff
@@ -20,6 +20,11 @@
 - Removed `FileRange` and `ClearRange`, they are now represented with `Azure::Core::Http::Range`.
 - Removed `Offset` and `Length` pair in options. They are now represented with `Azure::Core::Http::Range`.
 - Replace scoped enums that don't support bitwise operations with extensible enum.
+- Create APIs for Directory and File now returns `FileShareSmbProperties` that aggregates SMB related properties.
+
+### New Features
+
+- Added support for `CreateIfNotExists` for Share and Directory clients, and `DeleteIfExists` for Share, Directory and File clients.
```

Comment:
Does it make this PR not `single-purpose`?

Reply:
Sure, I'll remove the change.